docs/sources/installation/configuration.md

```diff
@@ -350,7 +350,7 @@ Default is `false`.
 
 ### data_source_proxy_whitelist
 
-Define a white list of allowed ips/domains to use in data sources. Format: `ip_or_domain:port` separated by spaces.
+Define a whitelist of allowed ips/domains to use in data sources. Format: `ip_or_domain:port` separated by spaces. This feature is currently not available for PostgreSQL, MySQL, and MSSQL data sources.
```

Comment:
How about this:

```suggestion
Define a whitelist of allowed IP addresses or domains to be used as data source URLs with the Grafana data source proxy. Format: `ip_or_domain:port` separated by spaces. PostgreSQL, MySQL, and MSSQL data sources  do not use this proxy and are therefore unaffected by this setting
```
I believe it is a little easier to read as it avoids repeating the `proxy` word and its variations 4 times in two sentences.